mercury.spec

```
@@ -45,7 +43,7 @@ Mercury devel
 
 %prep
 
-%autosetup -n mercury-%dl_version
+%autosetup -n mercury-%version
```

FWIW, I think this is the default.  Not reason enough to repush unless you want to, but:
```suggestion
%autosetup
```